src/types/Qualification.ts

```diff
@@ -0,0 +1,3 @@
+import { SelectChooseOption } from "./ChooseOption";
```

Comment:
Whats the purpos of this file? 

If we will have types in the future, we can always create a qualification types file

Reply:
we have types now, the future is here